omero-conda/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM ubuntu:14.04
 MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update -y && \
+        apt-get install -y --no-install-recommends \
```

Comment:
I had to do this once locally to get a JDK install to work too, but I thought more generally this was considered bad practice.